apps/web-client/src/routes/orders/suppliers/reconcile/[id]/+page.svelte

```diff
@@ -9,6 +9,14 @@
 	import Page from "$lib/components/Page.svelte";
 
 	import { view } from "@librocco/shared";
+	import type { PageData } from "./$types";
+	import { addOrderLinesToReconciliationOrder } from "$lib/db/cr-sqlite/order-reconciliation";
+	import { page } from "$app/stores";
+	import { onDestroy, onMount } from "svelte";
+	import { invalidate } from "$app/navigation";
+
+	// implement order reactivity/sync
+	export let data: PageData;
 	// Mock data for the comparison view
 	const mockSupplierBooks = [
```

Comment:
There is still mock data being used here. The UI displays the name & id of each supplier order in this reconciliation. We can either make updates to get this data, or challenge the mocks - maybe we don't need the name, we could just display a list of ids, and make them links so a user can easily navigate to and see the details of included supplier orders. We can make a call in this PR, or write up a small enhancement ticket